draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -154,6 +177,26 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
         persistent
       />
     </StoryWrapper.Row>
+    <StoryWrapper.Row rowTitle="Custom Content">
+      <GuidanceBlock
+        illustration={<Informative alt="" />}
+        content={<CustomContent />}
```

Comment:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/763385/209060013-3d948834-d31e-4ec3-8dbd-6016e797bef2.png">

Can we add a description text here for the new prop as well please, so when people look at it understand better when/how to use it. I think it has to be added to the `DefaultStory.argTypes` on line 78.
My idea would be something along these lines:
```
If you need to render custom content insight the `GuidanceBlock` use this prop instead of the default `text` option.
```